extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DevServicesConfig.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.oidc.runtime;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class DevServicesConfig {
+
+    /**
+     * If DevServices has been explicitly enabled or disabled. DevServices is generally enabled
+     * by default, unless there is an existing configuration present.
+     * <p>
+     * When DevServices is enabled Quarkus will attempt to automatically configure and start
+     * a database when running in Dev or Test mode and when Docker is running.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
+
+    /**
+     * The container image name to use, for container based DevServices providers.
+     */
+    @ConfigItem
+    public Optional<String> imageName;
+
+    /**
+     * The Keycloak realm path.
+     */
+    @ConfigItem
+    public Optional<String> realm;
```

Comment:
I think this property needs more explanation, for example is it a file system path, a classpath path, both?

Reply:
@geoand Yes, it was only a quick prototype, for the moment it is treated as a classpath resource but a file system path will be supported as well. For a while it will be me trying to figure how Dev Services work :-)

Reply:
Cool 😎

Reply:
@geoand I've prototyped the code to support a host system resource as well